core/templates/pages/skill-editor-page/navbar/skill-editor-navbar.directive.ts

```diff
@@ -92,7 +92,46 @@ angular.module('oppia').directive('skillEditorNavbar', [
             });
           };
 
+          $scope.toggleNavigationOptions = function() {
+            $scope.showNavigationOptions = !$scope.showNavigationOptions;
+          };
+          $scope.selectMainTab = function() {
+            $scope.activeTab = 'Editor';
```

Comment:
Can we use named constants for the activeTab strings?

Reply:
Done.